src/argilla/server/models.py

```diff
@@ -116,7 +123,7 @@ class Response(Base):
     def __repr__(self):
         return (
             f"Response(id={str(self.id)!r}, record_id={str(self.record_id)!r}, user_id={str(self.user_id)!r}, "
-            f"inserted_at={str(self.inserted_at)!r}, updated_at={str(self.updated_at)!r})"
+            f"status={self.status!r}, inserted_at={str(self.inserted_at)!r}, updated_at={str(self.updated_at)!r})"
```

Comment:
By using `self.status`, you will get the Enum object itself, which may not be the desired representation. Accessing `self.status.value` will return the corresponding value of the Enum member as a string.

```suggestion
            f"status={self.status.value!r}, inserted_at={str(self.inserted_at)!r}, updated_at={str(self.updated_at)!r})"
```

Reply:
Yes, I saw that, but I thought that it would be a good idea to see where the value is coming from. Updating this!

Reply:
Also, I've run into an issue with the `Enum` when using Python 3.11, because there is a [breaking change in Python 3.11](https://github.com/python/cpython/issues/100458) when using `Enum`s like `MyEnum(str, Enum): ...`

This is a problem because the endpoint generated here https://github.com/argilla-io/argilla/blob/5e3ecd7d0e8a99b7be8d4412d9f3f94410afce43/src/argilla/server/apis/v0/handlers/token_classification.py#L61 using Python 3.11 is `/api/datasets/{dataset_id}/TaskType.token_classification` instead of `/api/datasets/{dataset_id}/TokenClassification` and when calling the API from the client, for example to log some records in the server with `rg.log`, you will get a 404.

If supporting Python 3.11 is required then maybe this could be done:

```python
import sys
from enum import Enum

if sys.version_info >= (3, 11):
    from enum import StrEnum
else:
    class StrEnum(str, Enum):
        pass

class MyEnum(StrEnum):
    FOO = "foo"
    BAR = "bar"

print(f"{MyEnum.FOO}")
```

Reply:
I understand. We are currently using Python 3.7, so we may need to revisit this before upgrading to version 3.11.